feat(curriculum): adding teacher bot workshop (#55870)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b59b12a745e10011158f7b.md

---
id: 66b59b12a745e10011158f7b
title: Step 4
challengeType: 1
dashedName: step-4
---

# --description--

> The next step is to create a few more variables that you will use in future bot messages.

Reviewer: ```suggestion
The next step is to create a few more variables that will be used in future bot messages.
```
